test/integration/environment-template-integration.ts

```diff
@@ -126,10 +126,15 @@ describe('Environment template Api', () => {
     let installTemplate: InstallTemplate
 
     before(async () => {
-      space = (await createTestSpace(client, 'EnvironmentTemplate')) as Space
+      // Add random string to suite name to mitigate side effects
+      // with parallel running test.
+      // Slice to 22 characters to get not more then 30 chars in total.
+      const testSuitName = generateRandomId('EnvironmentTemplate').slice(0, 22)
```

Comment:
Just thinking if wouldn't be better to rather shorten the prefix instead of slicing the random part?